Kwartracker/Custom Components/Neumorphic Views/NeumorphicEffect.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  NeumorphicEffect.swift
+//  Kwartracker
+//
+//  Created by Leah Joy Ylaya on 5/28/21.
+//
+
+import SwiftUI
+
+struct NeumorphicEffect: View {
+    var rectSize: CGSize
+    var cornerRadius: CGFloat = 20
+    let backgroundColor: Color = Color(Asset.Colors.solitudeGrey.color)
+    let shadowOffset = CGPoint(x: 6, y: 6)
+    let blackOpacity: Double = 0.15
+    let position: CGFloat = 10
+    let shadowRadius: CGFloat = 5
+    let blurRadius: CGFloat = 15
+    let opacity: Double = 0.9
```

Comment:
Make this private if these variables are not accessible outside.

Reply:
https://github.com/jaimejazarenoiii/kwartracker-ios/pull/33/commits/0f5d5c5f3fba4bd729366bcf55789bc9e8405b3b